docs/api/browser-window.md

```diff
@@ -1208,12 +1208,19 @@ Returns `Boolean` - Whether the window's document has been edited.
 
 #### `win.capturePage([rect, ]callback)`
 
-* `rect` [Rectangle](structures/rectangle.md) (optional) - The bounds to capture
 * `callback` Function
   * `image` [NativeImage](native-image.md)
 
 Same as `webContents.capturePage([rect, ]callback)`.
 
+#### `win.capturePage([rect])`
+
+* `rect` [Rectangle](structures/rectangle.md) (optional) - The bounds to capture
+
+* Returns `Promise<NativeImage>` - Returns [NativeImage](native-image.md)
+
+Same as `webContents.capturePage(rect)`.
```

Comment:
Not sure what this is for, or what the "same as" in the original docs is for, maybe get rid of this and the original line?